add-on/panels/js/panel.jsx

```diff
@@ -745,6 +737,29 @@ loop.panel = (function(_, mozL10n) {
       );
     },
 
+    _renderRoomList: function() {
+      if (this.state.rooms.length) {
+        return (
+          <div className="room-list">{
+            this.state.rooms.map(function(room) {
+              if (this.state.openedRoom !== null &&
+                room.roomToken !== this.state.openedRoom) {
+                return null;
+                }
+
+              return (
+              <RoomEntry
+                dispatcher={this.props.dispatcher}
+                isOpenedRoom={room.roomToken === this.state.openedRoom}
+                key={room.roomToken}
+                room={room}/>
+                );
+              }, this)
```

Comment:
You have some odd indenting going on several places above that'll also need to be adjusted when flipping the condition.